src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOperators.cs

```diff
@@ -5349,5 +5349,39 @@ static void Main()
             string expectedOutput = @"1";
             CompileAndVerify(source, expectedOutput: expectedOutput);
         }
+
+
+        [Fact]
+        public void TestNullCoalesceWithDefault()
```

Comment:
>TestNullCoalesceWithDefault [](start = 20, length = 27)

Please add similar tests for a concrete user defined structure.

Reply:
Added a test.